<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8"/>
<style>body{background-color:white;}</style>
<script src="netvis_top5_files/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="netvis_top5_files/vis-9.1.0/vis-network.min.css" rel="stylesheet" />
<script src="netvis_top5_files/vis-9.1.0/vis-network.min.js"></script>
<script src="netvis_top5_files/visNetwork-binding-2.1.2/visNetwork.js"></script>
  <title>visNetwork</title>
</head>
<body>
<div id="htmlwidget_container">
  <div id="htmlwidget-5029fb68a7e18882bf6b" style="width:100%;height:1000px;" class="visNetwork html-widget"></div>
</div>
<script type="application/json" data-for="htmlwidget-5029fb68a7e18882bf6b">{"x":{"nodes":{"id":["KDM1A","RRM1","SREK1IP1","RFC1","ZNF483","RBBP5","MON1B","THAP5","C11orf58","SNW1","SMARCE1","TOPBP1","CIR1","LARP7","KRT222","OMG"],"label":["KDM1A","RRM1","SREK1IP1","RFC1","ZNF483","RBBP5","MON1B","THAP5","C11orf58","SNW1","SMARCE1","TOPBP1","CIR1","LARP7","KRT222","OMG"],"value":[1,2,1,2,3,1,1,2,5,5,1,1,1,2,3,1],"indegree":[0,2,0,2,0,1,1,2,1,1,1,1,1,2,0,1],"outdegree":[1,0,1,0,3,0,0,0,4,4,0,0,0,0,3,0],"title":["Lysine-specific histone demethylase 1A; Histone demethylase that demethylates both 'Lys-4'<br>(H3K4me) and 'Lys-9' (H3K9me) of histone H3, thereby acting as a coactivator or a<br>corepressor, depending on the context. Acts by oxidizing the substrate by FAD to generate<br>the corresponding imine that is subsequently hydrolyzed. Acts as a corepressor by mediating<br>demethylation of H3K4me, a specific tag for epigenetic transcriptional activation.<br>Demethylates both mono- (H3K4me1) and di-methylated (H3K4me2) H3K4me. May play a role in the<br>repression of neuronal genes. Alone, it is unable to demethyl [...]","Ribonucleoside-diphosphate reductase large subunit; Provides the precursors necessary for<br>DNA synthesis. Catalyzes the biosynthesis of deoxyribonucleotides from the corresponding<br>ribonucleotides","Protein SREK1IP1; Possible splicing regulator involved in the control of cellular survival","Replication factor C subunit 1; The elongation of primed DNA templates by DNA polymerase<br>delta and epsilon requires the action of the accessory proteins PCNA and activator 1. This<br>subunit binds to the primer-template junction. Binds the PO-B transcription element as well<br>as other GA rich DNA sequences. Could play a role in DNA transcription regulation as well as<br>DNA replication and/or repair. Can bind single- or double-stranded DNA; AAA ATPases","Zinc finger protein 483; May be involved in transcriptional regulation; SCAN domain<br>containing","Retinoblastoma-binding protein 5; In embryonic stem (ES) cells, plays a crucial role in the<br>differentiation potential, particularly along the neural lineage, regulating gene induction<br>and H3 'Lys-4' methylation at key developmental loci, including that mediated by retinoic<br>acid (By similarity). As part of the MLL1/MLL complex, involved in mono-, di- and<br>trimethylation at 'Lys-4' of histone H3. Histone H3 'Lys-4' methylation represents a<br>specific tag for epigenetic transcriptional activation; WD repeat domain containing","MON1 homolog B, secretory trafficking associated; Belongs to the MON1/SAND family","THAP domain-containing protein 5; Has sequence-specific DNA-binding activity and can<br>function as transcriptional repressor (in vitro). May be a regulator of cell cycle: THAP5<br>overexpression in human cell lines causes cell cycle arrest at G2/M phase; THAP domain<br>containing","Small acidic protein; Chromosome 11 open reading frame 58; Belongs to the SMAP family","SNW domain-containing protein 1; Involved in transcriptional regulation. Modulates TGF-<br>beta-mediated transcription via association with SMAD proteins, MYOD1-mediated transcription<br>via association with PABPN1, RB1- mediated transcriptional repression, and retinoid-X<br>receptor (RXR)- and vitamin D receptor (VDR)-dependent gene transcription in a cell<br>line-specific manner probably involving coactivators NCOA1 and GRIP1. Is involved in<br>NOTCH1-mediated transcriptional activation. Binds to multimerized forms of Notch<br>intracellular domain (NICD) and is proposed to recruit transcriptional coac [...]","SWI/SNF-related matrix-associated actin-dependent regulator of chromatin subfamily E member<br>1; Involved in transcriptional activation and repression of select genes by chromatin<br>remodeling (alteration of DNA-nucleosome topology). Component of SWI/SNF chromatin<br>remodeling complexes that carry out key enzymatic activities, changing chromatin structure<br>by altering DNA-histone contacts within a nucleosome in an ATP-dependent manner. Belongs to<br>the neural progenitors- specific chromatin remodeling complex (npBAF complex) and the<br>neuron-specific chromatin remodeling complex (nBAF complex). D [...]","DNA topoisomerase 2-binding protein 1; Required for DNA replication. Plays a role in the<br>rescue of stalled replication forks and checkpoint control. Binds double- stranded DNA<br>breaks and nicks as well as single-stranded DNA. Recruits the SWI/SNF chromatin remodeling<br>complex to E2F1- responsive promoters. Down-regulates E2F1 activity and inhibits<br>E2F1-dependent apoptosis during G1/S transition and after DNA damage. Induces a large<br>increase in the kinase activity of ATR","Corepressor interacting with RBPJ 1; May modulate splice site selection during alternative<br>splicing of pre-mRNAs (By similarity). Regulates transcription and acts as corepressor for<br>RBPJ. Recruits RBPJ to the Sin3-histone deacetylase complex (HDAC). Required for<br>RBPJ-mediated repression of transcription","La-related protein 7; Negative transcriptional regulator of polymerase II genes, acting by<br>means of the 7SK RNP system. Within the 7SK RNP complex, the positive transcription<br>elongation factor b (P-TEFb) is sequestered in an inactive form, preventing RNA polymerase<br>II phosphorylation and subsequent transcriptional elongation; La ribonucleoprotein domain<br>containing","Keratin-like protein KRT222; Keratins, type II","Oligodendrocyte-myelin glycoprotein; Cell adhesion molecule contributing to the interactive<br>process required for myelination in the central nervous system"],"modularity":["5","5","4","4","3","3","3","1","2","2","2","2","2","1","1","1"],"r2":[0,0.906284824092032,0,0.9095757323355,0,0.858224076276249,0.8727614353198671,0.944422832016414,0.882944954156569,0.882944954156569,0.886220744315389,0.893548846491063,0.912283705407277,0.890847698842551,0,0.839633499725609],"c2":[0.863694127514034,null,0.84107747528052,null,2.65858428726954,0,0,null,3.54577659485886,3.53768412471952,0,null,null,null,2.57467341324259,0],"hub":["FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","TRUE","TRUE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE"],"hub_class":[null,null,null,null,null,null,null,"responder","driver","generalhub",null,null,null,null,null,null],"tf":["FALSE","FALSE","FALSE","FALSE","TRUE","FALSE","FALSE","TRUE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE"],"group":["non-TF & non-hub","non-TF & non-hub","non-TF & non-hub","non-TF & non-hub","TF & non-hub","non-TF & non-hub","non-TF & non-hub","TF & non-hub","non-TF & driver","non-TF & generalhub","non-TF & non-hub","non-TF & non-hub","non-TF & non-hub","non-TF & non-hub","non-TF & non-hub","non-TF & non-hub"],"enrichment":["GO:RNA splicing(5/15&padj=0.00443735995881758),GO:response to ionizing radiation(3/15&padj=0.0145719540938919),GO:nucleobase metabolic process(2/15&padj=0.0201739839213974),GO:intrinsic apoptotic signaling pathway in response to DNA damage by p53 class mediator(2/15&padj=0.0217089152776518),GO:nuclear receptor binding(3/15&padj=0.00384698530691475),GO:nuclear androgen receptor binding(2/15&padj=0.00384698530691475),GO:transcription coactivator activity(3/15&padj=0.0177400988052074),GO:RNA polymerase II-specific DNA-binding transcription factor binding(3/15&padj=0.0239617244119233),GO:DNA-binding transcription factor binding(3/15&padj=0.0480091007641378),KEGG:Thermogenesis(2/9&padj=0.113875725167534),","GO:response to ionizing radiation(3/15&padj=0.0145719540938919),GO:DNA-templated DNA replication(3/15&padj=0.0183670391108158),GO:nucleobase metabolic process(2/15&padj=0.0201739839213974),GO:regulation of G0 to G1 transition(2/15&padj=0.0201739839213974),GO:G0 to G1 transition(2/15&padj=0.0201739839213974),GO:DNA replication(3/15&padj=0.0450612937767082),KEGG:Pyrimidine metabolism(1/9&padj=0.122460698208743),KEGG:Glutathione metabolism(1/9&padj=0.122460698208743),KEGG:Drug metabolism - other enzymes(1/9&padj=0.136284382159397),KEGG:Nucleotide metabolism(1/9&padj=0.136284382159397),KEGG:Purine metabolism(1/9&padj=0.160937407649885),","GO:RNA splicing(5/15&padj=0.00443735995881758),","GO:DNA-templated DNA replication(3/15&padj=0.0183670391108158),GO:DNA replication(3/15&padj=0.0450612937767082),KEGG:Mismatch repair(1/9&padj=0.113875725167534),KEGG:DNA replication(1/9&padj=0.122460698208743),KEGG:Base excision repair(1/9&padj=0.122460698208743),KEGG:Nucleotide excision repair(1/9&padj=0.122460698208743),","","KEGG:Cushing syndrome(1/9&padj=0.180435481356704),","GO:viral transcription(3/15&padj=0.00295270666202162),GO:viral gene expression(3/15&padj=0.00791969096254559),KEGG:Mitophagy - animal(1/9&padj=0.153963540520008),","","","GO:viral transcription(3/15&padj=0.00295270666202162),GO:RNA splicing(5/15&padj=0.00443735995881758),GO:viral gene expression(3/15&padj=0.00791969096254559),GO:intrinsic apoptotic signaling pathway in response to DNA damage by p53 class mediator(2/15&padj=0.0217089152776518),GO:nuclear receptor binding(3/15&padj=0.00384698530691475),GO:nuclear androgen receptor binding(2/15&padj=0.00384698530691475),GO:transcription coactivator activity(3/15&padj=0.0177400988052074),GO:RNA polymerase II-specific DNA-binding transcription factor binding(3/15&padj=0.0239617244119233),GO:DNA-binding transcription factor binding(3/15&padj=0.0480091007641378),KEGG:Notch signaling pathway(2/9&padj=0.0342813782854198),KEGG:Spliceosome(1/9&padj=0.206728107718983),KEGG:Epstein-Barr virus infection(2/9&padj=0.113875725167534),KEGG:Viral carcinogenesis(1/9&padj=0.206728107718983),","GO:regulation of G0 to G1 transition(2/15&padj=0.0201739839213974),GO:G0 to G1 transition(2/15&padj=0.0201739839213974),GO:nuclear receptor binding(3/15&padj=0.00384698530691475),GO:transcription coactivator activity(3/15&padj=0.0177400988052074),GO:RNA polymerase II-specific DNA-binding transcription factor binding(3/15&padj=0.0239617244119233),GO:DNA-binding transcription factor binding(3/15&padj=0.0480091007641378),KEGG:ATP-dependent chromatin remodeling(1/9&padj=0.158420397856949),KEGG:Hepatocellular carcinoma(1/9&padj=0.184969461101152),KEGG:Thermogenesis(2/9&padj=0.113875725167534),","GO:response to ionizing radiation(3/15&padj=0.0145719540938919),GO:DNA-templated DNA replication(3/15&padj=0.0183670391108158),GO:DNA replication(3/15&padj=0.0450612937767082),KEGG:Homologous recombination(1/9&padj=0.122460698208743),","GO:RNA splicing(5/15&padj=0.00443735995881758),KEGG:Notch signaling pathway(2/9&padj=0.0342813782854198),KEGG:Epstein-Barr virus infection(2/9&padj=0.113875725167534),","GO:viral transcription(3/15&padj=0.00295270666202162),GO:RNA splicing(5/15&padj=0.00443735995881758),GO:viral gene expression(3/15&padj=0.00791969096254559),","",""]},"edges":{"from":["KDM1A","SREK1IP1","ZNF483","ZNF483","ZNF483","C11orf58","C11orf58","C11orf58","C11orf58","SNW1","SNW1","SNW1","SNW1","KRT222","KRT222","KRT222"],"to":["RRM1","RFC1","RBBP5","MON1B","THAP5","RRM1","SNW1","SMARCE1","TOPBP1","RFC1","C11orf58","CIR1","LARP7","THAP5","LARP7","OMG"],"arrows.to.type":["arrow","arrow","arrow","arrow","arrow","arrow","arrow","arrow","arrow","arrow","arrow","arrow","arrow","arrow","arrow","arrow"]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot"},"manipulation":{"enabled":false},"interaction":{"hover":true,"zoomSpeed":1},"layout":{"randomSeed":1},"edges":{"arrows":"to"},"groups":{"non-TF & non-hub":{"color":"#97C2FC","shape":"dot"},"TF & non-hub":{"color":"#FF7F00","shape":"dot"},"non-TF & generalhub":{"color":"#9DCE03FF","shape":"square"},"TF & generalhub":{"color":"#FF7F00","shape":"square"},"non-TF & driver":{"color":"#9DCE03FF","shape":"triangle"},"TF & driver":{"color":"#FF7F00","shape":"triangle"},"non-TF & responder":{"color":"#9DCE03FF","shape":"triangleDown"},"TF & responder":{"color":"#FF7F00","shape":"triangleDown"},"non-TF & dual":{"color":"#9DCE03FF","shape":"diamond"},"useDefaultGroups":true,"TF & dual":{"color":"#FF7F00","shape":"diamond"}}},"groups":["non-TF & non-hub","TF & non-hub","non-TF & driver","non-TF & generalhub"],"width":"100%","height":"1000px","idselection":{"enabled":false,"style":"width: 150px; height: 26px","useLabels":true,"main":"Select by id"},"byselection":{"enabled":true,"style":"width: 150px; height: 26px","multiple":true,"hideColor":"rgba(200,200,200,0.5)","highlight":false,"variable":"enrichment","main":"Select by enrichment","values":["GO:DNA replication(3/15&padj=0.0450612937767082)","GO:DNA-binding transcription factor binding(3/15&padj=0.0480091007641378)","GO:DNA-templated DNA replication(3/15&padj=0.0183670391108158)","GO:G0 to G1 transition(2/15&padj=0.0201739839213974)","GO:intrinsic apoptotic signaling pathway in response to DNA damage by p53 class mediator(2/15&padj=0.0217089152776518)","GO:nuclear androgen receptor binding(2/15&padj=0.00384698530691475)","GO:nuclear receptor binding(3/15&padj=0.00384698530691475)","GO:nucleobase metabolic process(2/15&padj=0.0201739839213974)","GO:regulation of G0 to G1 transition(2/15&padj=0.0201739839213974)","GO:response to ionizing radiation(3/15&padj=0.0145719540938919)","GO:RNA polymerase II-specific DNA-binding transcription factor binding(3/15&padj=0.0239617244119233)","GO:RNA splicing(5/15&padj=0.00443735995881758)","GO:transcription coactivator activity(3/15&padj=0.0177400988052074)","GO:viral gene expression(3/15&padj=0.00791969096254559)","GO:viral transcription(3/15&padj=0.00295270666202162)","KEGG:ATP-dependent chromatin remodeling(1/9&padj=0.158420397856949)","KEGG:Base excision repair(1/9&padj=0.122460698208743)","KEGG:Cushing syndrome(1/9&padj=0.180435481356704)","KEGG:DNA replication(1/9&padj=0.122460698208743)","KEGG:Drug metabolism - other enzymes(1/9&padj=0.136284382159397)","KEGG:Epstein-Barr virus infection(2/9&padj=0.113875725167534)","KEGG:Glutathione metabolism(1/9&padj=0.122460698208743)","KEGG:Hepatocellular carcinoma(1/9&padj=0.184969461101152)","KEGG:Homologous recombination(1/9&padj=0.122460698208743)","KEGG:Mismatch repair(1/9&padj=0.113875725167534)","KEGG:Mitophagy - animal(1/9&padj=0.153963540520008)","KEGG:Notch signaling pathway(2/9&padj=0.0342813782854198)","KEGG:Nucleotide excision repair(1/9&padj=0.122460698208743)","KEGG:Nucleotide metabolism(1/9&padj=0.136284382159397)","KEGG:Purine metabolism(1/9&padj=0.160937407649885)","KEGG:Pyrimidine metabolism(1/9&padj=0.122460698208743)","KEGG:Spliceosome(1/9&padj=0.206728107718983)","KEGG:Thermogenesis(2/9&padj=0.113875725167534)","KEGG:Viral carcinogenesis(1/9&padj=0.206728107718983)"]},"main":{"text":"excitatory sub-network: top 5 --selected genes","style":"font-family:Georgia, Times New Roman, Times, serif;font-weight:bold;font-size:20px;text-align:center;"},"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)","tooltipStay":300,"tooltipStyle":"position: fixed;visibility:hidden;padding: 5px;white-space: nowrap;font-family: verdana;font-size:14px;font-color:#000000;background-color: #f5f4ed;-moz-border-radius: 3px;-webkit-border-radius: 3px;border-radius: 3px;border: 1px solid #808074;box-shadow: 3px 3px 10px rgba(0, 0, 0, 0.2);","highlight":{"enabled":true,"hoverNearest":true,"degree":2,"algorithm":"all","hideColor":"rgba(200,200,200,0.5)","labelOnly":true},"collapse":{"enabled":false,"fit":false,"resetHighlight":true,"clusterOptions":null,"keepCoord":true,"labelSuffix":"(cluster)"}},"evals":[],"jsHooks":[]}</script>
<script type="application/htmlwidget-sizing" data-for="htmlwidget-5029fb68a7e18882bf6b">{"viewer":{"width":"100%","height":"1000px","padding":15,"fill":false},"browser":{"width":"100%","height":"1000px","padding":40,"fill":false}}</script>
</body>
</html>
